src/comm_manager.cpp

```diff
@@ -403,6 +412,49 @@ void CommManager::heartbeat_callback(void)
   this->send_heartbeat();
 }
 
+void CommManager::config_set_callback(uint8_t device, uint8_t configuration)
+{
+  uint8_t requested_device{device};
+  if(device >=Configuration::DEVICE_COUNT)
+    device = Configuration::DEVICE_COUNT;
+  ConfigManager::config_response resp = RF_.config_manager_.attempt_set_configuration(static_cast<device_t>(device), configuration);
+  comm_link_.send_config_status(sysid_, requested_device, resp.successful, resp.reboot_required, resp.message);
+}
```

Comment:
We probably want to disallow set configurations while the vehicle is armed

Reply:
I added this logic to ConfigManager.